updated c_trace_de to support other than web.ctrace.de urls

5ila5 opened the pull request:
I updated c_trace_de to also allow other subdomains (see #697) and formats because there where only a few changes to be made  and I personally think it would not make sense to make a new integration, there are a few more like this one out there (googling: `site:apps.c-trace.de`)

but I'm not sure if this are to many possible arguments. (especially ical_url_file feels kind of janky)  I'm open to changes or other Idears

mampfes commented:
I also don't really like the extra `ical_url_file` argument. My proposal would be to add a mapping table which maps the service name to the ical file name. And I would also remove the `web.` from Landau (maybe using the same mapping table).

5ila5 commented:
- I added a Mapping table
- added all apps.c-trace.de municipalities in the mapping table
-  and added all municipalities I could find by googling into the table in the md file (apps.`...` und web.`...`)

mampfes commented:
Great. Thanks for your contribution.
